scripts/publish.sh

```diff
@@ -14,23 +14,29 @@ function bump_and_npm_publish {
         echo "Publishing '$1@$2'"
         npm version "$2" -w "$1"
         npm publish --access public -w "$1"
+        echo "Published '$1@$2'"
     fi
 }
 
 GIT_ROOT_DIR=$(git rev-parse --show-toplevel)
 VERSION=$1
+BY_PASS_VERSION_CHECK=$2
 
 # ensure version is of format x.y.z
-if [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
-    echo "VERSION '$VERSION' is not of format x.y.z"
-    exit 1
+if [[ "$BY_PASS_VERSION_CHECK" != "true" ]]; then
+    # ensure version is of format x.y.z
+    if [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
See my comment above. I would use the following regex `^([0-9]+\.[0-9]+\.[0-9]+|0\.0\.1-[0-9a-fA-F]{40})$` and not have a bypass